src/pages/index.tsx

```diff
@@ -390,38 +250,50 @@ const cardsOnlyProduct = [
       "/card-Sleeves/card-Sleeve_Teal.avif",
       "/card-Sleeves/card-Sleeve_Hazel.avif",
     ],
+    imageAlt: [
+      "/card-Sleeves/card-Sleeve-Black_Alt.avif",
+      "/card-Sleeves/card-Sleeve_Ocean_Alt.avif",
+      "/card-Sleeves/card-Sleeve_Gray_Alt.avif",
+      "/card-Sleeves/card-Sleeve_Teal_Alt.avif",
+      "/card-Sleeves/card-Sleeve_Hazel_Alt.avif",
+    ]
   },
   {
     name: "Card Sleeve",
     colours: ["black"],
     price: "C$69",
     image: ["/card-Sleeves/card-Sleeve-Black_Ash.avif"],
+    imageAlt: ["/card-Sleeves/card-Sleeve-Black_Ash_Alt.avif"],
```

Comment:
All the image alts from here on out needs to be fixed. Including the ones that are empty